features/enable_fips_pro.feature

```diff
@@ -140,12 +142,13 @@ Feature: FIPS enablement in PRO cloud based machines
             fips-updates  +yes +disabled +FIPS compliant crypto packages with stable security updates
             """
         When I run `pro enable <fips-service> --assume-yes` with sudo
-        Then stdout matches regexp:
+        Then stdout contains substr:
             """
             Updating <fips-name> package lists
             Installing <fips-name> packages
+            Updating standard Ubuntu package lists
```

Comment:
I think this line needs to be added for all of the messages that are touched in this PR